src/main/java/com/shevtsov/dao/ClientDao.java

```diff
@@ -3,5 +3,41 @@
 import com.shevtsov.domain.Client;
 
 public interface ClientDao {
+
+    /**
+     * Saves Client-object to DB.
```

Comment:
In general this method could save Client not in DB, so you can use some common description like save to storage

Reply:
Right. Done.

Reply:
Where?